examples/resources/plugins/teleport-slack.toml

```diff
@@ -1,16 +1,40 @@
-# example slack plugin configuration TOML file
+# Copied from https://github.com/gravitational/teleport-plugins/blob/master/access/slack/example_config.toml
+
+# Example Slack plugin configuration TOML file
 
 [teleport]
-addr = "auth.example.com:3025"           # Teleport Auth Server GRPC API address
-client_key = "/var/lib/teleport/plugins/slack/auth.key" # Teleport GRPC client secret key
-client_crt = "/var/lib/teleport/plugins/slack/auth.crt" # Teleport GRPC client certificate
-root_cas = "/var/lib/teleport/plugins/slack/auth.cas"   # Teleport cluster CA certs
+# This section 
+
+# Teleport Auth/Proxy Server address.
+# addr = "example.com:3025"
+#
+# Should be port 3025 for Auth Server and 3080 or 443 for Proxy.
+# For Teleport Cloud, should be in the form "your-account.teleport.sh:443".
+
+# Credentials generated with `tctl auth sign`.
+#
+# When using --format=file:
+# identity = "/var/lib/teleport/plugins/slack/auth_id"    # Identity file
+#
+# When using --format=tls:
+# client_key = "/var/lib/teleport/plugins/slack/auth.key" # Teleport TLS secret key
+# client_crt = "/var/lib/teleport/plugins/slack/auth.crt" # Teleport TLS certificate
+# root_cas = "/var/lib/teleport/plugins/slack/auth.cas"   # Teleport CA certs
 
 [slack]
-token = "xoxb-11xx"             # Slack Bot OAuth token
-# Optional Slack Rooms
-recipients = ["team","devops"]
+# Slack Bot OAuth token
+token = "xoxb-11xx"
```

Comment:
We should add that we support reading the token from a file
Related: https://github.com/gravitational/teleport-plugins/issues/505

Reply:
Not sure if we should add it here, in the doc itself or both

Reply:
I've added this in a comment in the example file